db/migrate/20220706130805_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter
+
```

Comment:
- [ ] Please add a default value to the `posts_counter` so that when a new user is created, they have 0 posts. You can do it this way 👇 
```
t.integer :posts_counter, default: 0
```